### ISSUE TITLE
As an SP I want to specify which groups are required for my service

### ISSUE THREAD

**Issue by venekamp** [SURFscz/SBS#511]
My service requires a number of groups to operate. When a CO is linked to my service, SRAM must populate the CO with my predefined groups. As an SP, I need a method of defining these groups in SRAM.

As an example, let's take Delena, it needs several (5) groups:
- delena_login
- delena_cpu
- user
- data
- sw

It's up to the CO admin to put users in those groups. These groups are roles for Delena and if users are not a member of delena_login, they cannot not log in. Without this, the CO admin must add these groups by hand for each CO he/she want to link to my service.

**Comment by oharsta** [comment]
@venekamp @baszoetekouw Can this issue be merged with https://github.com/SURFscz/SBS/issues/437?

**Comment by venekamp** [comment]
Ja... was al opzoek gegaan naar wat nu de status is van service groups, maar was even vergeten hoe die heten. Directe aanleiding was dat er voor een project (C-SCALE) een nieuwe CO aangemaakt is. Het sync script dat tussen SRAM en de CUA (LDAP voor een aantal SURF research diensten) loopt, begon te klagen dat er voor _die_ CO groepen ontbraken. Toen ik er geen issue over vinden kan, dacht ik dat schiet ik er eentje in. @oharsta dank voor je scherpe blik.

**Comment by FlorisFokkinga** [comment]
Duplicate.
